frontend/src/authentication/LoginPage.jsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import Cookies from "js-cookie";
+
+function LoginPage() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+
+  const BACKEND_URL = import.meta.env.VITE_BACKEND_URL;
+
+  const navigate = useNavigate();
+
+  const handleLogin = async (event) => {
+    event.preventDefault();
+    setError("");
+
+    try {
+      const response = await fetch(`${BACKEND_URL}/auth/login`, {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({ username, password }),
+        credentials: "include",
+      });
+
+      if (!response.ok) {
+        const errorResponse = await response.json();
+        throw new Error(
+          errorResponse.message || "Error logging in. Refresh and try again"
```

Comment:
Since the backend knows if the error was user not found or wrong password, you could consider showing these errors to the user when appropriate, to give them more info

Reply:
Done.